docs/package.json

```diff
@@ -79,6 +79,7 @@
 		"@playwright/test": "^1.49.0",
 		"@rushstack/node-core-library": "^5.9.0",
 		"@types/node": "^22.9.1",
+		"@wayneferrao/docusaurus-search-local": "^0.48.5",
```

Comment:
Presumably we should remove the old `@easyops-cn/docusaurus-search-local` dependency too?